internal/extproc/translator/gemini_helper.go

```diff
@@ -514,6 +516,10 @@ func geminiFinishReasonToOpenAI(reason genai.FinishReason) openai.ChatCompletion
 		return openai.ChatCompletionChoicesFinishReasonStop
 	case genai.FinishReasonMaxTokens:
 		return openai.ChatCompletionChoicesFinishReasonLength
+	case "":
+		// For intermediate chunks in a streaming response, the finish reason is an empty string.
```

Comment:
is this documented or it's just a finding? if it is documented, let's put a link